artic-ncov2019/2.0.0/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM ubuntu:xenial
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL container.version="1"
```

Comment:
could you please change this to `LABEL dockerfile.version="1"` ?

Reply:
Added!